ERCS/erc-7580.md

```diff
@@ -0,0 +1,92 @@
+---
+eip: 7580
+title: Inter dapp tracking inferface
+description: Share an inter-dapp tracking inferface for dapp interactions and support for promotion/ads solution
```

Comment:
Neither your description or title provides a lot of information about your proposal. Most ERCs define standards for dapps to interface/interact together. Try to include details in these fields that differentiates your proposal from others.